src/barkhaPJun20/CountDigitsLettersUpperLowerSpecial.java

```diff
@@ -0,0 +1,69 @@
+package barkhaPJun20;
+
+public class CountDigitsLettersUpperLowerSpecial {
+	
+	void countDigitsInString(String name) {
+		int count=0;
+		for(int index=0; index<name.length(); index++) {
+			char ch=name.charAt(index);
+			if (Character.isDigit(ch))
+					count++;
+		}
+		System.out.println("Number of DIGITS in Predefined String are: "+count);
+		
+	}
+	void countLettersInString(String name) {
+		int count=0;
+		for(int index=0; index<name.length(); index++) {
+			char ch=name.charAt(index);
+			if (Character.isLetter(ch))
+					count++;
+		}
+		System.out.println("Number of LETTERS in Predefined String are: "+count);
+		
+		
+	}
+	void countUpperCaseInString(String name) {
+		int count=0;
+		for(int index=0; index<name.length(); index++) {
+			char ch=name.charAt(index);
+			if (Character.isUpperCase(ch))
+					count++;
+		}
+		System.out.println("Number of UPPER CASE LETTERS in Predefined String are: "+count);
+		
+	}
+	void countLowerCaseInString(String name) {
+		int count=0;
+		for(int index=0; index<name.length(); index++) {
+			char ch=name.charAt(index);
+			if (Character.isLowerCase(ch))
+					count++;
+		}
+		System.out.println("Number of LOWER CASE LETTERS in Predefined String are: "+count);
+		
+	}
+	void countSpecialCharInString(String name) {
```

Comment:
Barkha, my expectation is to write a single method in which all conditions should be checked. and in that method only expected output should be printed.

Reply:
Wrote in if else loop...please review.